send_message_msteams_remove_quote_duplicates.py

```diff
@@ -0,0 +1,302 @@
+                                        quote_author TEXT NOT NULL,
+                                        send_dt TIMESTAMP NOT NULL,
+                                        picture_url TEXT NOT NULL,
+                                        image_aws_s3_key TEXT NOT NULL
+                                    )
+                                """)
+                self.conn.commit()
+                logging.info("Ensured the quotes table exists.")
+
+        except Exception as e:  # TODO: think about particular exceptions
+            logging.error(f"Failed to ensure the quotes table exists: {e}")
+
+    def check_quote_duplicates(self, quote_text, quote_author):
+        try:
+            with self.conn.cursor() as cursor:
+                select_query = """
+                            SELECT * FROM daily_quotes
+                            WHERE quote = %s AND quote_author = %s
+                            LIMIT 1
+                        """
+                cursor.execute(select_query, (quote_text, quote_author))
+                record = cursor.fetchone()
+                if record:
+                    logging.info("Duplicate quote found.")
+                    return record
+                else:
+                    logging.info("No duplicate quote found.")
+                    return None
+
+        except Exception as e:  # TODO: think about particular exceptions
+            logging.error(f"Failed to perform duplicated check: {e}")
+
+    def load_quote(self, quote_text, quote_author, send_dt, image_url, s3_key):
+        try:
+            with self.conn.cursor() as cursor:
+                insert_query = sql.SQL("""
+                    INSERT INTO daily_quotes (
+                        quote, 
+                        quote_author, 
+                        send_dt, 
+                        picture_url, 
+                        image_aws_s3_key
+                    )
+                    VALUES (%s, %s, %s, %s, %s)
+                """)
+                cursor.execute(
+                    insert_query,
+                    (
+                        quote_text,
+                        quote_author,
+                        send_dt,
+                        image_url,
+                        s3_key
+                    )
+                )
+                self.conn.commit()
+                logging.info("Quote successfully saved to RDS.")
+        except Exception as e:  # TODO: think about particular exceptions
+            logging.error(f"Failed to save quote to RDS: {e}")
+
+
+class MessageSenderExecutor:
+    def __init__(self, quote_generator, image_generator, message_sender, s3_loader, rds_loader):
+        self.quote_generator = quote_generator
+        self.image_generator = image_generator
+        self.message_sender = message_sender
+        self.s3_loader = s3_loader
+        self.rds_loader = rds_loader
+
+    def run_process(self):
+        while True:
```

Comment:
It's better to avoid while True statements, not to run inside an infinite loop. Try to restrict the number of calls and fail in case no other quotes came to the mind